<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableWidgetItem</class>
 <widget class="QWidget" name="TableWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetOutside" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	font: 14pt &quot;黑体&quot;;	
	color: rgb(255,200,74);
}

QWidget#titleWidget{
	background-color: rgba(38,105,148,209);
	border-left: 1px solid rgba(44,179,229,255);
	border-bottom: 1px solid rgba(14,57,83,255);
}
/*
QWidget#titleWidget::hover{
	background-color: rgb(255, 85, 0);
	border-left: 1px solid rgba(255,179,229,,255);
	border-bottom: 1px solid rgba(255,57,83,255);
}*/
/*
QWidget#titleWidget::pressed{
	background-color: rgba(0,0,0,209);
	border-left: 1px solid rgba(0,0,0,255);
	border-bottom: 1px solid rgba(0,0,0,255);
}
*/
QWidget#contentWidget{
	border-left: 3px solid rgba(44,179,229,255);
	border-bottom: 3px solid rgba(14,57,83,255);
}

QPushButton#pb1{		
	border-image: url(:/images/item/info.png);
}
/*
QPushButton#pb1::hover{			
	border-image: url(:/images/item/info_hover.png);
}

QPushButton#pb1::checked{		
	border-image: url(:/images/item/info_checked.png);
}
*/
QPushButton#pb2{	
	border-image: url(:/images/item/add.png);
}
QPushButton#pb2::hover{			
	border-image: url(:/images/item/add_hover.png);
}
/*
QPushButton#pb2::checked{		
	border-image: url(:/images/item/add_checked.png);
}
*/
QPushButton#pb3{
	border-image: url(:/images/item/sub.png);
}
QPushButton#pb3::hover{
	border-image: url(:/images/item/sub_hover.png);
}
/*
QPushButton#pb3::checked{
	border-image: url(:/images/item/sub_checked.png);
}
*/</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="titleWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="9">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>14</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>12</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="iconLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>12</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="textLabel">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>599</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="5">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="pb2">
           <property name="toolTip">
            <string>添加</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="8">
          <widget class="QPushButton" name="pb3">
           <property name="toolTip">
            <string>删除</string>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="contentWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
